x-pack/test_serverless/api_integration/test_suites/common/spaces.ts

```diff
@@ -13,24 +13,206 @@ export default function ({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
 
   describe('spaces', function () {
-    it('rejects request to create a space', async () => {
-      const { body, status } = await supertest
-        .post('/api/spaces/space')
-        .set(svlCommonApi.getInternalRequestHeader())
-        .send({
-          id: 'custom',
-          name: 'Custom',
-          disabledFeatures: [],
+    describe('route access', () => {
+      describe('disabled', () => {
+        it('#delete', async () => {
+          const { body, status } = await supertest
+            .delete('/api/spaces/space/default')
+            .set(svlCommonApi.getCommonRequestHeader());
+
+          // normally we'd get a 400 bad request if we tried to delete the default space
+          svlCommonApi.assertApiNotFound(body, status);
         });
 
-      // in a non-serverless environment this would succeed with a 200
-      expect(body).toEqual({
-        statusCode: 400,
-        error: 'Bad Request',
-        message:
-          'Unable to create Space, this exceeds the maximum number of spaces set by the xpack.spaces.maxSpaces setting',
+        it('#create', async () => {
+          const { body, status } = await supertest
+            .post('/api/spaces/space')
+            .set(svlCommonApi.getCommonRequestHeader())
+            .send({
+              id: 'custom',
+              name: 'Custom',
+              disabledFeatures: [],
+            });
+
+          svlCommonApi.assertApiNotFound(body, status);
+        });
+
+        it('#copyToSpace', async () => {
+          const { body, status } = await supertest
+            .post('/api/spaces/_copy_saved_objects')
+            .set(svlCommonApi.getCommonRequestHeader());
+
+          // without a request body we would normally a 400 bad request if the endpoint was registered
```

Comment:
This was the first round of tests I wrote. Is it worth supplying valid request bodies? I shouldn't make any difference based on how we're validating the result.

Reply:
>Is it worth supplying valid request bodies? 

I don't think so, what you have is more than enough, IMO.